File: Hardwaver/lib/config/app_config.dart
class AppConfig {
  static const String appName = '滴墨讲';

  static const String openaiBaseUrl = 'https://api.openai.com/v1';
  static const String defaultModel = 'gpt-4o-mini';
  static const String defaultEmbeddingModel = 'text-embedding-3-small';
  static const String defaultWhisperModel = 'whisper-1';
  static const String defaultTtsModel = 'tts-1';

  static const int shortTermMaxMessages = 20;
  static const int longTermTopK = 5;
  static const double longTermSimilarityThreshold = 0.3;
  static const int midTermRecallLimit = 3;
  static const Duration sessionTimeout = Duration(minutes: 30);
}

File: DimoTalk.Maui/Config/AppConfig.cs
namespace DimoTalk.Maui.Config;

public static class AppConfig
{
    public const string AppName = "滴墨讲";

    public const string DefaultModel = "gpt-4o-mini";
    public const string DefaultEmbeddingModel = "text-embedding-3-small";
    public const string DefaultWhisperModel = "whisper-1";
    public const string DefaultTtsModel = "tts-1";

    public const int ShortTermMaxMessages = 20;
    public const int LongTermTopK = 5;
    public const double LongTermSimilarityThreshold = 0.3;
    public const int MidTermRecallLimit = 3;
    public static readonly TimeSpan SessionTimeout = TimeSpan.FromMinutes(30);
}
